### PR TITLE
Make some strings unhardcoded so they can be translated

### PR DESCRIPTION
**Pull request by Abstructor**
Title

### CODE REVIEW

**Comment by lexisother** [comment]
???

**Comment by Abstructor** [comment]
Whoops. I might have accidentally closed this PR while checking on it

File: src/Powercord/coremods/updater/index.js [hunk 1]
```diff
@@ -359,7 +359,7 @@ module.exports = async () => {
 
   powercord.api.settings.registerSettings('pc-updater', {
     category: 'pc-updater',
-    label: 'Updater', // Note to self: add this string to i18n last :^)
+    label: Messages.REPLUGGED_UPDATES_UPDATER, // Note to self: add this string to i18n last :^)
```
Comment (asportnoy) [inline src/Powercord/coremods/updater/index.js:362]:
```suggestion
    label: Messages.REPLUGGED_UPDATES_UPDATER,
```

**Review by asportnoy — CHANGES_REQUESTED** [review]
There are some more strings that should be added to i18n as well (only including ones in settings and updater for now)
https://github.com/replugged-org/replugged/blob/c8089068d15d2cffd19248a14ce3356877aa535b/src/Powercord/coremods/settings/components/ErrorBoundary.jsx#L66-L72
https://github.com/replugged-org/replugged/blob/c8089068d15d2cffd19248a14ce3356877aa535b/src/Powercord/coremods/settings/components/GeneralSettings/index.jsx#L37
https://github.com/replugged-org/replugged/blob/c8089068d15d2cffd19248a14ce3356877aa535b/src/Powercord/coremods/updater/index.js#L369
https://github.com/replugged-org/replugged/blob/c8089068d15d2cffd19248a14ce3356877aa535b/src/Powercord/coremods/updater/components/Settings.jsx#L350-L433
https://github.com/replugged-org/replugged/blob/c8089068d15d2cffd19248a14ce3356877aa535b/src/Powercord/coremods/updater/components/Settings.jsx#L439

File: src/Powercord/coremods/updater/index.js (hunk 1 above)
Comment (Abstructor) [reply src/Powercord/coremods/updater/index.js]: Changes resolved.

**Review by asportnoy — APPROVED** [review]
LGTM

**Review by lexisother — APPROVED** [review]
Yop.

**Review by 12944qwerty — CHANGES_REQUESTED** [review]
Several, if not all, of the badges' tooltips could be included. [coremods/badges](https://github.com/Abstructor/replugged/blob/main/src/Powercord/coremods/badges/index.js#L55-L81)
This [notice](https://github.com/Abstructor/replugged/blob/main/src/Powercord/coremods/notices/index.js#L43) can probably be translated
I think some toasts could also be translated, but not 100% sure if we need to/should. [coremods/routelinks](https://github.com/Abstructor/replugged/blob/main/src/Powercord/coremods/routelinks/index.js)

I've checked most files, but be sure to check the components folder as well just in case.

**Comment by asportnoy** [comment]
The badges already have translations.

**Comment by Abstructor** [comment]
Should I make the built in coremods commands descriptions (like this one below) translatable too?
image

**Comment by Abstructor** [comment]
And strings like these are pretty much impossible to create an i18n string for since they have a variable in the middle of the string
image

**Comment by asportnoy** [comment]
```json
MY_STRING: "Something {varname}"
```

```js
Messages.MY_STRING.format({
  varname: "some value"
});
```

**Comment by Abstructor** [comment]
Ah, thank you

**Comment by Abstructor** [comment]
That should be it I think.

**Comment by Abstructor** [comment]
Done

**Review by lleyton — APPROVED** [review]
Looks good with me, I'll need local validation though

**Review by 12944qwerty — APPROVED** [review]

**Pull request merged**